scripts/solana_verify.ts

```diff
@@ -0,0 +1,106 @@
+/*
+  This script fetchs a key/value pair and its proofs from the Umbrella API and
+  validated them with a call to the `chain` Solana program. The Merkel root hash
+  is already stored in a block which is a Solana data account referenced with a
+  Program Derived Account (kind of public-key), this last can is derived in our
+  program using a the block ID and the official program ID of `chain`.
+ */
+
+import * as anchor from '@project-serum/anchor';
+import { Program } from '@project-serum/anchor';
+import { PublicKey, SystemProgram } from '@solana/web3.js';
+import { LeafKeyCoder, LeafValueCoder } from '@umb-network/toolbox';
+import axios from 'axios';
+import 'dotenv/config';
+
+const {API_BASE_URL, API_KEY} = process.env;
+
+// program ID on Sandbox/Mainnet
+const chainId = "4SPgs3L7Ey9VyRuZwx4X3y86LSAZXP2Hhpz9Sps4v3iT";
+
+const IDL = JSON.parse(
+  require("fs").readFileSync("./artifacts/solana-idl/chain.json", "utf8")
+);
+
+const main = async() => {
+
+  const provider = anchor.AnchorProvider.env();
+  anchor.setProvider(provider);
+
+  let program = new Program(
+    IDL,
+    new PublicKey(chainId),
+    provider
+  );
+
+  // We fetch the last block ID from the API
+  const latest_block_id: number = await axios
+    .get(`${API_BASE_URL}/blocks/latest?chainId=solana`, {
+      headers: {
+        authorization: API_KEY
+      }
+    })
+    .then(({data}) => data.data.blockId)
+    .catch((err) => console.log(`error in fetching last block: ${err}`));
+
+  const seed = LeafValueCoder.encode(latest_block_id, '');
+  let pair = "ETH-USD"
+
+  const data = await axios
+    .get(`${API_BASE_URL}/proofs?keys[]=${pair}&chainId=solana`, {
+      headers: {
+        authorization: API_KEY
+      }
+    })
+    .then(({data}) => data.data.leaves[0])
+    .catch((err) => console.log(`error in fetching last block: ${err}`));
```

Comment:
This console is the same as L44. What do you think here should be:

```suggestion
    .catch((err) => console.log(`error in fetching proofs from ETH-USD pair: ${err}`));
```